src/richText.js

```diff
@@ -22,51 +22,203 @@
  *
  * Any other HTML has undefined behavior
  */
+
+(function () {
+    // set CKEditor base path before loading ckeditor
+    var require = window.requirejs,
+        path = require.toUrl("vellum/../lib/ckeditor/").replace(/\?.*/, "");
+    window.CKEDITOR_BASEPATH = path;
```

Comment:
Tested locally in standalone mode (built and not) and in local HQ with `VELLUM_DEBUG` set to each of `dev`, `dev-min`, and not set.